tests/pipeline/test_pipeline.py

```diff
@@ -2,29 +2,40 @@
 
 
 def test_dryrun(demo_dir):
-    r = run(["snakemake", "-n"], dir_path=demo_dir)
+    r = run("snakemake -n --cores 1", dir_path=demo_dir)
     message = "This was a dry-run (flag -n). The order of jobs does not reflect the order of execution."
     assert message in r.stdout
 
+def test_pipeline_no_run(demo_dir):
+    LOGGER.info("run entire pipeline with \'run: false\'")
+
+    # change all "run: true" config values to "run: false" and save result into new config file
+    # runthe pipeline with each module turned off
+    run("sed 's/run: true/run: false/g' config.yaml > config_norun.yaml  ",demo_dir)
+    pipeline_run = run(["snakemake",  f"-j{CORES}", "--configfile", "config_norun.yaml"], demo_dir)
```

Comment:
Does this test make sense here? Everything should be done anyway. So we should rather turn it around and test first if I can switch on a module and then run it.

Reply:
Same goes for the other parts.